### PR TITLE
[K32W0] SVE fixes

### PR DESCRIPTION
**Pull request by doru91**
Main fixes/functionalities brought in this PR:
* add initial enablement for K32W041A/AM
* add linker file fixes for OTA enablement
* add detokenizer support for the elock and
moved the script location to a generic location
* move tinycrypt implementation in a separate class
* enable low power with SDK 2.6.6
* enable MACv2
* refactoring around PDM storage
* add size-optimized BLE libraries

Signed-off-by: Doru Gucea <doru-cristian.gucea@nxp.com>
Signed-off-by: Marius Tache <marius.tache@nxp.com>
Signed-off-by: Andrei Menzopol <andrei.menzopol@nxp.com>

### CODE REVIEW

**Review by woody-apple — APPROVED** [review]
Fast tracking platform changes

**Comment by andy31415** [comment]
`2022-07-08 06:26:04 INFO    NXP_K32W0_SDK_ROOT is not set`

**Comment by woody-apple** [comment]
@doru91 Please note the build failures here

**Comment by doru91** [comment]
> @doru91 Please note the build failures here

@woody-apple , the build fails because it doesn't find the Docker container. I guess that someone with Docker administrator rights has to push the new K32W0 Docker image to Docker - this is required after https://github.com/project-chip/connectedhomeip/pull/20472 was recently merged. cc @andy31415

File: integrations/docker/images/chip-build-vscode/Dockerfile [hunk 1]
```diff
@@ -75,7 +75,7 @@ ENV IDF_PATH=/opt/espressif/esp-idf/
 ENV IDF_TOOLS_PATH=/opt/espressif/tools
 ENV IMX_SDK_ROOT=/opt/fsl-imx-xwayland/5.10-hardknott/
 ENV NRF5_TOOLS_ROOT=/opt/NordicSemiconductor/nRF5_tools
-ENV NXP_K32W061_SDK_ROOT=/opt/sdk/sdks
+ENV NXP_K32W0_SDK_ROOT=/opt/sdk
```
Comment (andy31415) [inline integrations/docker/images/chip-build-vscode/Dockerfile:78]: Changing in dockerfiles have to be created in a separate PR and then version has to be updated.

If we are building 0.5.87 today then the env is not the same and actually vscode image will NOT be a ble to build the current PR.

Comment (andy31415) [reply integrations/docker/images/chip-build-vscode/Dockerfile:78]: I think we need this as a prerequisite so that vscode image can also compile things (all developers use vscode image)

Comment (doru91) [reply integrations/docker/images/chip-build-vscode/Dockerfile]: thanks, understood, will submit the changes from the vscode dockerfile in a separate PR.

**Review by andy31415 — APPROVED** [review]
